### ISSUE TITLE
My recommendation: support common markdown

### ISSUE THREAD

**Issue by Trevoke** [pothibo/marked.js#1]
I think that's a really good effort and we need this spec to be adopted globally. Right now there's too many interpretations of the original markdown, because it's so vague.

http://commonmark.org/

**Comment by pothibo** [comment]
Yes! That’s what I meant when I said Stack Overflow’s markdown. I just couldn’t remember the name of it!

I’ll add it in the README thanks!

Pier-Olivier Thibault
pothibo@gmail.com
